scripts/build-release.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -eu
+
+./mvnw -B -ntp install jib:dockerBuild -Pflatten
```

Comment:
you can remove `-Pflatten` - that was something specific for the multi-module planetiler repo.

Reply:
Adjusted.